From 88c544ae031d77172fb6ce438e5f2f145c49e6c9 Mon Sep 17 00:00:00 2001
From: Edmunt Pienkowsky <roed@onet.eu>
Date: Sat, 11 Jan 2020 19:28:14 +0100
Subject: [PATCH] Increase response timeouts

Current timeouts are too short. On my Raspberry Pi Zero timeouts are very common.

diff --git a/temper.py b/temper.py
index f129951..13b86dd 100755
--- a/temper.py
+++ b/temper.py
@@ -146,7 +146,7 @@ class USBRead(object):
     os.write(fd, struct.pack('8B', 0x01, 0x86, 0xff, 0x01, 0, 0, 0, 0))
     firmware = b''
     while True:
-      r, _, _ = select.select([fd], [], [], 0.1)
+      r, _, _ = select.select([fd], [], [], 2)
       if fd not in r:
         break
       data = os.read(fd, 8)
@@ -162,7 +162,7 @@ class USBRead(object):
     os.write(fd, struct.pack('8B', 0x01, 0x80, 0x33, 0x01, 0, 0, 0, 0))
     bytes = b''
     while True:
-      r, _, _ = select.select([fd], [], [], 0.1)
+      r, _, _ = select.select([fd], [], [], 2)
       if fd not in r:
         break
       data = os.read(fd, 8)
-- 
2.24.1

